_pages/en_US/Mocha CFW (Haxchi).txt

```diff
@@ -76,7 +76,7 @@ You can use the (Home) button to exit the Homebrew Launcher if you need to purch
   + Use syshax.xml (coldboothax): **off**
 11. Press (A) to confirm and exit
 12. Your Wii U should display a black screen; this is the RedNAND setup
-13. Press the power button **on the console** (not the gamepad) to start the copy of your internal system memory to your SD card
+13. Press the power button **on the console** (not the gamepad) to start copying your internal system memory to your SD card
```

Comment:
Whoops, I'll revert that then.

I'm a little confused about what it means to "start" a copy. Is it like "starting up"?